classes/citation/job/pid/Arxiv.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file classes/citation/job/pid/Arxiv.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Arxiv
+ *
+ * @ingroup citation
+ *
+ * @brief Arxiv class
+ */
+
+namespace PKP\citation\job\pid;
+
+class Arxiv extends BasePid
+{
+    /** @copydoc AbstractPid::regex */
+    public const string regex = '%\b(([\w-]+://?|www[.])[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|/)))%s';
```

Comment:
Since 3.5.0-x uses a minimum PHP version of 8.2, typed constants should not be used as they're only allowed since 8.3.